### ISSUE TITLE
JSON validation

### ISSUE THREAD

**Issue by jhalterman** [elkozmon/zoonavigator#55]
ZooNavigator actively validates JSON data when editing a ZNode, but I think it would be good to go a step further and provide either a popup warning when someone tries to save invalid JSON or just not allow doing that.

**Comment by elkozmon** [comment]
True, a popup warning would be nice. I can't tell you when, but I'd be happy to add this.
